src/libraries/apiclient/apiclientcore.js

```diff
@@ -1530,6 +1530,15 @@ define(["events", "appStorage"], function(events, appStorage) {
             type: "POST",
             url: url
         })
+    }, ApiClient.prototype.sendSyncplayCommand = function (sessionId, command, options) {
+        if (!sessionId) throw new Error("null sessionId");
+        if (!command) throw new Error("null command");
+        var url = this.getUrl("Syncplay/" + sessionId + "/" + command, options || {});
+        this.ensureWebSocket();
+        return this.ajax({
+            type: "POST",
+            url: url
+        })
```

Comment:
We are also extremely close to migrating our API client to a [separate repository](https://github.com/jellyfin/jellyfin-apiclient-javascript) so it can be used in other projects, courtesy of @MrTimscampi. It should be much easier to work with than the mess we have here.

Reply:
I definitely missed that repository, ops. I'll look into it.

Reply:
Don't worry about it :smile: we haven't exactly been advertising the change yet.

Reply:
It's still being heavily messed with since it's not 100% compatible with the web client yet...

No ETA on the move though, so definitely keep it in there :) Once your PR here is merged, feel free to open one over there to add it, or I'll port it there myself if you don't have time ;)

Reply:
No worries, plenty of time on my end as well at the moment :wink:
I'll keep polishing it here then.